Khan/wonder-blocks
# [feat:dropdown] WB-615: OptionItem -> Add ARIA props and switch to LabelMedium

## Summary
**OptionItem:**
- Added `aria-label` prop to the `OptionItem` component. This change is introduced to add the ability to pass any relevant information on every option item that screen readers can understand.
- Changed `font-size` from `bold` to `normal` to match design specs.

## Test plan
- Open https://deploy-preview-565--wonder-blocks.netlify.com/#multiselect.
- Check the `accessibility` example and check that `aria-label` is added to the option items.

![option-item-label](https://user-images.githubusercontent.com/843075/67599604-708f8e00-f73e-11e9-9b39-8953d064eeab.gif)

cc @aag
